### PR TITLE
Feature/sprint 3.0

### PR DESCRIPTION
**Pull request by DariaDostoevskaia**

### CODE REVIEW

**Comment by DariaDostoevskaia** [comment]
not ready

File: Assets/Scripts/EnemyShip.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    [SerializeField] private List<KeyCode> _upArrows;
+    [SerializeField] private List<KeyCode> _downArrows;
+    [SerializeField] private List<KeyCode> _leftArrows;
+    [SerializeField] private List<KeyCode> _rightArrows;
+
+    private float _currentSpeed;
+    private Vector3 _lastMovement;
+    private Vector3 _enemyMousePosition;
+
+    private Vector3 _movement;
```
Comment (Overlordshor) [inline Assets/Scripts/EnemyShip.cs]: Why is it in the enemy ship class?
The task was to create a ship player 2, which will be an ally of player 1

File: Assets/Scripts/EnemyShip.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+    private void Start()
+    {
+    }
+
+    private void Update()
+    {
+        //if (Input.GetKey(KeyCode.L))
+        //    transform.rotation *= Quaternion.Euler(0f, 360f * Time.deltaTime, 0f);
```
Comment (Overlordshor) [inline Assets/Scripts/EnemyShip.cs]: commented code should not be in a pull request

File: Assets/Scripts/EnemyShip.cs [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    //    {
+    //        _spriteRenderer.flipX = true;
+    //        _isForward = true;
+    //    }
+    //    else
+    //    {
+    //        _spriteRenderer.flipX = false;
+    //        _isForward = false;
+    //    }
+    //}
```
Comment (Overlordshor) [inline Assets/Scripts/EnemyShip.cs]: remove

File: Assets/Scripts/Sample.cs [hunk 4]
```diff
@@ -19,6 +18,12 @@ private void Start()
         {
             Debug.Log(items[i]);
         }
+
+        var positions = new List<Vector3>(20);
+        var number = Random.Range(0, 10);
+        for (int i = 0; i < 20; i++)
+        {
+        }
```
Comment (Overlordshor) [inline Assets/Scripts/Sample.cs:26]: Missing:
1. Generating random vectors
2. Adding elements to the list.
3. Vector addition
4. Removal of vectors.

**Review by Overlordshor — CHANGES_REQUESTED** [review]

**Pull request closed without merging**